src/lib/rmt/wizard_maria_db_page.rb

```diff
@@ -20,6 +20,11 @@
 require 'rmt/maria_db/new_root_password_dialog'
 require 'rmt/execute'
 require 'ui/event_dispatcher'
+begin
+  require 'yast2/systemd/service'
+rescue LoadError
+  Yast.import 'SystemdService'
+end
```

Comment:
Could you add a comment to the code explaining why the begin/rescue logic is needed? Thanks!

Reply:
I've added a comment which I expect it clarifies things.

Reply:
Thanks!